analytics/views/realm_activity.py

```diff
@@ -62,6 +66,12 @@ def get_count(user_summary: Dict[str, Dict[str, str]], k: str) -> str:
         else:
             return ""
 
+    def get_bot_owner_email(user_summary: Dict[str, Dict[str, str]]) -> mark_safe:
+        email = user_summary["bot_owner_email"]
+        bot_owner = user_summary["bot_owner"]
+        email_link = f'<a href="mailto:{email}">{bot_owner}</a>'
+        return mark_safe(email_link)
```

Comment:
Is this really "safe"?  That is, if my owner name is `</a><a href="script:alert('!')">Alex`, what happens?

I suspect we need to escape these before we insert them.  See [`format_html` instead of `mark_safe`](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.html.format_html).